x-pack/plugins/triggers_actions_ui/public/application/sections/alert_details/components/alert_details.tsx

```diff
@@ -82,6 +90,28 @@ export const AlertDetails: React.FunctionComponent<AlertDetailsProps> = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  // Determine if any attached action has an issue with its connector
+  useEffect(() => {
+    (async () => {
+      let loadedConnectors: ActionConnector[] = [];
+      try {
+        loadedConnectors = await loadConnectors({ http });
```

Comment:
This approach seems to be fine to get this info. Just wonder if we can use Event Log instead... Is it possible? And if it is, maybe more efficient? 
There could be a one thing to worry about - a size of the response, but seems like we do not expect to have such a big number of connectors.

Reply:
Action execution failures are stored as `provider:action` in the event log which I don't think we're currently querying to get some of this information, so we would have to create new queries. I think this approach is ok for now because this is not blocking any other part of the page on this.